aeron-client/src/main/c/aeronc.h

```diff
@@ -491,14 +491,39 @@ int aeron_async_add_counter(
     size_t label_buffer_length);
 
 /**
- * Poll the completion of the aeron_async_add_counter call.
+ * Poll the completion of the `aeron_async_add_counter` or `aeron_async_add_static_counter` calls.
```

Comment:
Do back ticks work for doc formatting here?

Reply:
Should probably go with `<code>text</code>` to be consistent with Javadoc.

Reply:
Done.